apps/private-law/prl-cos/ithc.yaml

```diff
@@ -36,7 +36,7 @@ spec:
       readinessTimeout: 45
       readinessPeriod: 15
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/prl/cos:prod-1625208-20230901083551 #{"$imagepolicy": "flux-system:prl-cos"}
+      image: hmctspublic.azurecr.io/prl/cos:pr-1598-b7d9e5b-20230901102214 #{"$imagepolicy": "flux-system:prl-cos"}
```

Comment:
change the image policy to ithc-prl-cos-api

Reply:
@AlokDatta  updated policy to ithc-prl-cos-api